sktime/forecasting/base/_fh.py

```diff
@@ -273,23 +273,23 @@ def to_relative(self, cutoff=None):
             absolute = self.to_pandas()
             _check_cutoff(cutoff, absolute)
 
-            if isinstance(absolute, pd.DatetimeIndex):
-                # We cannot use the freq from the the ForecastingHorizon itself (or its
-                # wrapped pd.DatetimeIndex) because it may be none for non-regular
-                # indices, so instead we use the freq of cutoff.
-                freq = _get_freq(cutoff)
+            # We cannot use the freq from the ForecastingHorizon itself (or its
+            # wrapped pd.DatetimeIndex) because it may be none for non-regular
+            # indices, so instead we use the freq of cutoff.
+            freq = _get_freq(cutoff)
```

Comment:
does cutoff always have a frequency?

Reply:
I wonder too. There are not enough tests to say for sure. Not all public methods are tested explicitly, but only implicitly through forecasters and transformers.

Reply:
hm... somewhat troubling.
We may need an explicit design document on the different values that `cutoff` can (and is expected to) have; as well as behaviour of `ForecastingHorizon` around that. There is clearly an interaction with the `BaseForecaster`.